cmd/ef_tests/levm/runner/revm_runner.rs

```diff
@@ -255,9 +261,38 @@ pub fn ensure_post_state(
 }
 
 pub fn compare_levm_revm_account_updates(
+    test: &EFTest,
     levm_account_updates: &[AccountUpdate],
     revm_account_updates: &[AccountUpdate],
 ) -> AccountUpdatesReport {
+    let mut initial_accounts: HashMap<Address, Account> = test
+        .pre
+        .0
+        .iter()
+        .map(|(account_address, pre_state_value)| {
+            let account_storage = pre_state_value
+                .storage
+                .iter()
+                .map(|(key, value)| {
+                    let mut temp = [0u8; 32];
+                    key.to_big_endian(&mut temp);
+                    let storage_slot = StorageSlot {
+                        original_value: *value,
+                        current_value: *value,
+                    };
+                    (H256::from_slice(&temp), storage_slot)
+                })
+                .collect();
+            let account = Account::new(
+                pre_state_value.balance,
+                pre_state_value.code.clone(),
+                pre_state_value.nonce.as_u64(),
+                account_storage,
+            );
+            (*account_address, account)
+        })
+        .collect();
+    initial_accounts.insert(test.env.current_coinbase, Account::default());
```

Comment:
Wouldn't this bring issues if the coinbase account was already in the initial accounts? For example in a test that has the coinbase address in the pre state

Reply:
It wouldn't because we only use this for report printing and not for the main logic. The idea here is to include the coinbase account's initial state (which if not set in the pre-state data is the same as having an empty account) because it is being updated in the execution but we do not have track of its initial state.

Reply:
Done in [ea24a96](https://github.com/lambdaclass/ethrex/pull/1280/commits/ea24a96eb04ad376c45fa72ad4d8d1164915b446)